### PR TITLE
Use TestContainer Cloud on CI for windows tests that can not host the requested images

### PR DESCRIPTION
**Pull request by Mpdreamz**

### CODE REVIEW

**Comment by amannocci** [comment]
@Mpdreamz It seems like we are ready to move forward with windows tests using testcontainers cloud.

**Review by gregkalapos — APPROVED** [review]
@amannocci pinged me about this, PR looks ok to me. @stevejgordon could you please also look at this, before we merge?

**Review by Mpdreamz — COMMENTED** [review]
LGTM, will merge once all test changes pass. 

@gregkalapos mind giving this a LGTM? since i amended this commit we need another person to sign off 😄

File: .github/workflows/test-windows.yml
Comment (v1v) [inline .github/workflows/test-windows.yml:47]: I'm afraid this won't work for forked PRs. Shall this be split by enabling only when working on the upstream rather than on forked-PRs?

Comment (Mpdreamz) [reply .github/workflows/test-windows.yml:47]: That would be great @v1v 👍

Comment (v1v) [reply .github/workflows/test-windows.yml:47]: Cool, I'll code something. Then likely, I might need your help with skipping those tests for forked PRs.

**Pull request merged**
